openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/bigQueryConnection.json

```diff
@@ -30,6 +30,12 @@
       "$ref": "#/definitions/bigqueryType",
       "default": "BigQuery"
     },
+    "projectId": {
+      "title": "Project ID",
+      "description": "GCP Project ID to parse metadata from",
+      "type": "string",
+      "default": null
+    },
```

Comment:
Moving this to openmetadata-spec/src/main/resources/json/schema/security/credentials/gcpCredentials.json under gcpCredentialsPath would be more appropriate to maintain scope of the json schema property